core/libretro/libretro.cpp

```diff
@@ -353,11 +365,19 @@ void retro_set_environment(retro_environment_t cb)
       },
       {
          "reicast_cable_type",
+#ifdef LOW_END
+         "Cable type; VGA (RGB)|TV (RGB)|TV (Composite)",	
+#else
          "Cable type; TV (RGB)|TV (Composite)|VGA (RGB)",
+#endif
       },
       {
          "reicast_broadcast",
-         "Broadcast; PAL_M|PAL_N|NTSC|PAL|Default",
+#ifdef LOW_END
+         "Broadcast; Default|PAL_M|PAL_N|NTSC|PAL",
+#else
+         "Broadcast; PAL_M|PAL_N|NTSC|PAL|Default",	
+#endif
```

Comment:
I think `Default` should be the default setting for all platforms. I'm not sure why `PAL_M` is currently used instead.

Reply:
I will do this global.